tidb-cloud/config-s3-and-gcs-access.md

```diff
@@ -99,35 +99,47 @@ To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the
 
 ## Configure GCS access
 
-To allow TiDB cloud to access the source data in your GCS bucket, you need to configure the GCS access for each TiDB Cloud as a service on the GCP project and GCS bucket pair. Once the configuration is done for one cluster in a project, all database clusters in that project can access the GCS bucket.
+To allow TiDB Cloud to access the source data in your GCS bucket, you need to configure the GCS access for each TiDB Cloud as a service on the GCP project and GCS bucket pair. Once the configuration is done for one cluster in a project, all database clusters in that project can access the GCS bucket.
```

Comment:
”one cluster“ is better to modify to ”one TiDB Cluster“
”all database clusters“ is better to modify to ”all TiDB Clusters“